include/zephyr/audio/codec.h

```diff
@@ -34,56 +34,66 @@ extern "C" {
  * PCM audio sample rates
  */
 typedef enum {
-	AUDIO_PCM_RATE_8K	= 8000,		/**< 8 kHz sample rate */
-	AUDIO_PCM_RATE_16K	= 16000,	/**< 16 kHz sample rate */
-	AUDIO_PCM_RATE_24K	= 24000,	/**< 24 kHz sample rate */
-	AUDIO_PCM_RATE_32K	= 32000,	/**< 32 kHz sample rate */
-	AUDIO_PCM_RATE_44P1K	= 44100,	/**< 44.1 kHz sample rate */
-	AUDIO_PCM_RATE_48K	= 48000,	/**< 48 kHz sample rate */
-	AUDIO_PCM_RATE_96K	= 96000,	/**< 96 kHz sample rate */
-	AUDIO_PCM_RATE_192K	= 192000,	/**< 192 kHz sample rate */
+	AUDIO_PCM_RATE_8K = 8000,     /**< 8 kHz sample rate */
+	AUDIO_PCM_RATE_11K = 11025,   /**< 11.025 kHz sample rate*/
+	AUDIO_PCM_RATE_16K = 16000,   /**< 16 kHz sample rate */
+	AUDIO_PCM_RATE_22K = 22050,   /**< 22.05 kHz sample rate*/
+	AUDIO_PCM_RATE_24K = 24000,   /**< 24 kHz sample rate */
+	AUDIO_PCM_RATE_32K = 32000,   /**< 32 kHz sample rate */
+	AUDIO_PCM_RATE_44P1K = 44100, /**< 44.1 kHz sample rate */
+	AUDIO_PCM_RATE_48K = 48000,   /**< 48 kHz sample rate */
+	AUDIO_PCM_RATE_96K = 96000,   /**< 96 kHz sample rate */
+	AUDIO_PCM_RATE_192K = 192000, /**< 192 kHz sample rate */
 } audio_pcm_rate_t;
 
```

Comment:
I don't undertand this change? You seem to add some tabs / space. This seems not to be related with the subject of the patch.  api: codec: Extend API 

If you want to do some cleanups then it should be a separate patch. Otherwise, this doesn't bring any value as it is in this patch.

One patch should mostly do one thing!

Reply:
I just invoked `clang-format` as recommended by contribution guidelines (https://docs.zephyrproject.org/latest/contribute/guidelines.html#clang-format) to take care of code formatting. Is this something that shouldn't be done when changing existing files?

Reply:
> I just invoked `clang-format` as recommended by contribution guidelines (https://docs.zephyrproject.org/latest/contribute/guidelines.html#clang-format) to take care of code formatting. Is this something that shouldn't be done when changing existing files?

yes, this shouldn't be taken into consideration for existing code. Only for new code added. If you still want to make the change you can make it as a separate PR. 

But maybe that is a task after you get this merged.